README.md

```diff
@@ -85,18 +85,26 @@ Python versions can be changed with the `pyenv local` or `pyenv global` commands
 
 #### Python & dependencies
 
-Inside the project directory install python version, upgrade pip and install dependencies:
+Inside the project directory install python version, upgrade pip:
 
 ``` shell
 pyenv install
-pip install --upgrade pip setuptools pipenv
-pipenv install --dev
+pip install --upgrade pip setuptools
 ```
 
-###### NOTE: You may encounter an issue with the `python-snappy` installation with `pipenv` above on Mac OSX. If so, install with the recommended [workaround command](https://github.com/andrix/python-snappy#frequently-asked-questions) instead:
-``` shell
-CPPFLAGS="-I/usr/local/include -L/usr/local/lib" pipenv install --dev
-```
+Install poetry, poetry dotenv plugin and install dependencies:
+
+''' shell
+curl -sSL https://install.python-poetry.org | python3 - 
```

Comment:
We should pin the poetry version. We will have to update it somewhat frequently. Some lock files generated from specific versions cannot be used by other version. Given we pin the version in actions and Dockerfile, we should pin here to ensure we generate the lock using the same version. 

Use the latest version `1.8.2`

Reply:
Updated all ares to now use `v1.8.2` and Readme updated to install pinned version of Poetry